README.md

```diff
@@ -43,6 +52,70 @@ Commands:
 
 To generate a script, you'll need to set an `ADO_PAT` as an environment variable. Performing any of the commands that touch GitHub will need the `GH_PAT` environment variable.
 
+### Running a Migration 
+
+Covering running a migration from **Azure DevOps** to **GitHub.com**. 
+
+#### Download and Configure the GH-GEI command-line Tool
+
+Navigate to the `Releases` for this repo and grab the latest release for your local operating system.
+![Releases](https://user-images.githubusercontent.com/29484535/145065021-35f37a00-1a25-42a4-804d-11fd9f8cc811.png)
+Once you have downloaded the `Release`, you need to extract it to your local machine.
+**Note** you will want to place the `octoshift` executable somewhere easy to reference or add to your path.
+![Folder View](https://user-images.githubusercontent.com/29484535/145065026-a519a7f0-fc1d-46a1-a1a5-cd96743b1bd1.png)
+
+* [Linux add folder to path](https://linuxize.com/post/how-to-add-directory-to-path-in-linux/)
+* [Powershell add folder to path](https://stackoverflow.com/questions/714877/setting-windows-powershell-environment-variables/714918)
+
+Once you have pathed the tooling, you will need to set `2` environment variables. 
+
+* One will be called `GH_PAT` and will be your **GitHub Personal Access Token**
+* The other will be called `ADO_PAT` and will be your **Azure DevOps Access Token** 
+
+The scope needed for each token will depend on what command(s) you want run. See the scenarios below to ensure you have properly scoped personal access tokens. It's recommended that you pick the scenario which fits the needs of everything you want to do as part of migrating. 
+
+#### I just want to run some migrations and grant/revoke the migrator role
+Create a GitHub and Azure DevOps personal access tokens with the scope defined in GEI's [documentation](https://docs.github.com/en/early-access/github/migrating-with-github-enterprise-importer/migrating-to-github-enterprise-cloud-with-the-importer#step-2-assign-the-migration-permissions-role-and-ensure-the-migrator-has-the-expected-pat-scopes). This will allow you to run these commands:
+
+* generate-script (--repos-only)
+* migrate-repo
+* grant/revoke-migrator-role
+* create-team
+* add-team-to-repo
+* configure-autolink
+
+#### I want to do the above and also lock & disable the repo being migrated from Azure DevOps
+In order to use the following pre & post migration commands:
+
+* lock-ado-repo
+* disable-ado-repo
```

Comment:
this command also requires Code (Read, write & manage)

I think the GEI docs only require Code (Read)